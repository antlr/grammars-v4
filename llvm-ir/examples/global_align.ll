@g = global i32 0, align 1, section "foo"
@h = global i32 0, section "foo", align 1
