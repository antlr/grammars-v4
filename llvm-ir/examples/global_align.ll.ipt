compilationUnit
 topLevelEntity
  globalDef
   @g
   =
   immutable
    global
   type
    intType
     i32
   constant
    intConst
     0
   ,
   globalField
    align
     align
     1
   ,
   globalField
    section
     section
     "foo"
 topLevelEntity
  globalDef
   @h
   =
   immutable
    global
   type
    intType
     i32
   constant
    intConst
     0
   ,
   globalField
    section
     section
     "foo"
   ,
   globalField
    align
     align
     1
 <EOF>
