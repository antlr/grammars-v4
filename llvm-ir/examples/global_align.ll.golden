@g = global i32 0, section "foo", align 1
@h = global i32 0, section "foo", align 1
